chainer/training/updaters/multiprocess_parallel_updater.py

```diff
@@ -0,0 +1,376 @@
+import multiprocessing
+
+import six
+
+import chainer
+from chainer import cuda
+from chainer.dataset import convert
+from chainer.training.updater import StandardUpdater
+from chainer import variable
+
+try:
+    from cupy.cuda import nccl
+    _available = True
+except ImportError:
+    _available = False
+
+import numpy
+
+
+class _Worker(multiprocessing.Process):
+
+    def __init__(self, proc_id, pipe, master):
+        super(_Worker, self).__init__()
+        self.proc_id = proc_id
+        self.pipe = pipe
+        self.converter = master.converter
+        self.model = master._master
+        self.device = master._devices[proc_id]
+        self.iterator = master._mpu_iterators[proc_id]
+        self.n_devices = len(master._devices)
+
+    def setup(self):
+        print("setup %d" % self.device)
```

Comment:
Print message here seems unclear for users. How about explain what is happening here more clearly, or just removing this print message if it is actually unnecessary?